### DISCUSSION TITLE
How to display book order in a series (e.g., "1 of 5")?

### DISCUSSION THREAD

**Discussion by vawaver** [advplyr/audiobookshelf#3635] [Q&A]
Hello,

I am trying to organize my audiobooks in Audiobookshelf and would like to display the book order within a series, such as "1 of 5" for the first book in a series. However, I am unable to get this functionality to work.

Here is what I have tried so far:

1. I created a `series.json` file in the parent directory of the series with the following content:
   ```json
   {
     "name": "Démonská séria",
     "books": [
       {
         "title": "Tetovaný",
         "index": 1,
         "total": 5
       },
       {
         "title": "Pouštní kopí",
         "index": 2,
         "total": 5
       }
     ]
   }
   ```
   - This didn't seem to be recognized by Audiobookshelf.

2. I also attempted to add metadata directly to each book using a `metadata.json` file:
   ```json
   {
     "title": "Tetovaný",
     "author": "Peter V. Brett",
     "series": "Démonská séria",
     "seriesIndex": 1,
     "totalInSeries": 5,
     "description": "..."
   }
   ```
   - Audiobookshelf scanned the library but did not display the series or order information.

3. I tried adding a `metadata.opf` file in the book directory:
   ```xml
   <package xmlns="http://www.idpf.org/2007/opf" version="2.0">
     <metadata xmlns:dc="http://purl.org/dc/elements/1.1/">
       <dc:title>Tetovaný</dc:title>
       <dc:creator>Peter V. Brett</dc:creator>
       <meta name="series" content="Démonská séria"/>
       <meta name="series_index" content="1"/>
       <meta name="total_in_series" content="5"/>
     </metadata>
   </package>
   ```
   - This also did not work.

4. I checked the documentation and community forums but couldn’t find a definitive guide on how to achieve this.

I am running the latest version of Audiobookshelf via Docker. Could you please clarify:
- How to properly set up series information and book order so that Audiobookshelf recognizes and displays it?
- Whether this feature is fully supported or requires additional configuration?

Any idea how to solve?

**Comment by nichwall** [comment]
ABS just stored the series index of the book, it does not store the total number of books in the series (as part of the book metadata)

**Comment by vawaver** [comment]
It’s clear that Audiobookshelf currently only stores the series index of a book and does not include the total number of books in a series. Adding support for displaying the total number of books in a series would significantly enhance the functionality.

For example, instead of showing only "1" for the first book, the display could be "1 of 5." This would provide better context and make it easier to track progress within a series, especially for users managing extensive libraries.

Could this functionality be considered for a future update? 
If more specific examples or use cases are needed, I can provide them.

**Reply by advplyr** [reply]
The issue is that we don't enforce sequences to be numbers. I would like to implement that but there has been some pushback on users wanting to use letters

**Comment by vawaver** [comment]
I partially solved the problem using the tag `Subtitle`.
